projects/1.19/assets/quark/quark/lang/zh_cn.json

```
@@ -167,6 +174,136 @@
   "quark.jei.influence": "矩阵附魔影响",
   "quark.jei.boost_influence": "提高几率：",
   "quark.jei.dampen_influence": "降低几率：",
+  "quark.jei.hint_preamble": "[夸克]\n",
+
+  "quark.jei.hint.pickarang": "这是一把镐头回旋镖！可以抛出并采集它击中的东西，还能将掉落物带回来。也可以给它附魔，以获得更好的性能。",
+  "quark.jei.hint.flamerang": "用起来类似皮卡镖，但它拾取的任何物品在回程中都会免疫火焰伤害。",
+  "quark.jei.hint.echorang": "用起来类似皮卡镖，但会发出持续的振动，而且听到幽匿尖啸体的呼啸声后会延长飞行时间。",
+  "quark.jei.hint.abacus": "对一个方块使用，它会计算你看向的方块与前述方块之间的距离（三个坐标轴相加），最多48格。",
+  "quark.jei.hint.ancient_fruit": "砍倒远古树时掉落。食用后你会增加少量经验。",
+  "quark.jei.hint.ancient_sapling": "可以在远古城市的箱子中找到。",
+  "quark.jei.hint.backpack": "背上它可以获得额外的三行物品栏空间。",
+  "quark.jei.hint.blackstone_furnace": "如果位于这个熔炉下面的方块在点燃时会产生灵魂火，那么熔炉产生的也是灵魂火。",
+  "quark.jei.hint.blaze_lantern": "就像下界岩一样永久燃烧。",
```

```suggestion
  "quark.jei.hint.blaze_lantern": "类似下界岩，会使其上火焰永久燃烧。",
```

似乎原文就不怎么好，永久燃烧最好点明主体是火。
下几行的charcoal_block同理

---

思考了一会儿，想到了火的本质或者说定义是什么、燃烧的主语是什么……
扯远了，我平常说燃烧一般是xx燃烧（xx是燃料或者说可燃物），至于火在燃烧这种没有可燃物的表达……挺陌生的。当然，在MC中，火虽然不是燃料，但它也确实是一种方块，而不是所谓现象
然后我看了wiki，用的是火焰燃烧，只能顺从wiki

---

~~能把火焰改成火吗~~

---

"类似下界岩，会使其上的火永久燃烧。",
这样？我读起来很怪

---

这样呢？
```suggestion
  "quark.jei.hint.blaze_lantern": "类似下界岩，顶部点燃的火能够永久燃烧。",
```

---

合理，就这个吧